hadoop-ozone/dev-support/intellij/runConfigurations/Datanode2.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Datanode2" type="Application" factoryName="Application">
+    <option name="ALTERNATIVE_JRE_PATH" value="13" />
```

Comment:
Some setup may not have JRE 13. Should we keep JDK8 as a minimum supported Java version that is common on Mac Dev box?

Reply:
Good point. Those lines seem to be optional. I removed them...